### PR TITLE
Adds default Japanese fonts to fallback

### PR DESCRIPTION
**Pull request by Bhammy**
Adds a default Japanese font to the fallback font variable chain, so that we can give browsers _some_ direction.

### CODE REVIEW

**Comment by hallee** [comment]
I'd be interested in including `-apple-system` first; Helvetica Neue is no longer the default system font on Apple platforms and I believe Hiragino Sans has also been replaced by a Japanese variant of their San Fransisco font.

**Comment by Bhammy** [comment]
> -apple-system



> I'd be interested in including `-apple-system` first; Helvetica Neue is no longer the default system font on Apple platforms and I believe Hiragino Sans has also been replaced by a Japanese variant of their San Fransisco font.

As in,
`$font-default-sans-fallback: -apple-system, 'Helvetica Neue', Helvetica, 'Hiragino Sans', sans-serif;` ?

**Comment by hallee** [comment]
> > I'd be interested in including `-apple-system` first; Helvetica Neue is no longer the default system font on Apple platforms and I believe Hiragino Sans has also been replaced by a Japanese variant of their San Fransisco font.
> 
> As in,
> `$font-default-sans-fallback: -apple-system, 'Helvetica Neue', Helvetica, 'Hiragino Sans', sans-serif;` ?

Yep! And same for serif fallback. But I'm very bad at CSS and there might be a good reason not to use `-apple-system` that I'm not aware of

File: scss/fonts.scss [hunk 1]
```diff
@@ -239,8 +239,8 @@
  * non-system fonts are only applied once loaded in order to
  * prevent FOIT.
  */
-$font-default-sans-fallback: 'Helvetica Neue', Helvetica, sans-serif;
-$font-default-serif-fallback: Georgia, serif;
+ $font-default-sans-fallback: -apple-system, 'Helvetica Neue', Helvetica, 'Hiragino Sans', sans-serif;
+ $font-default-serif-fallback: -apple-system, Georgia, 'Hiragino Sans', serif;
```
Comment (hallee) [inline scss/fonts.scss:243]:
```suggestion
$font-default-serif-fallback: Georgia, 'Hiragino Sans', serif;
```

actually I think `-apple-system` is explicitly the sans-serif version

File: scss/fonts.scss [hunk 2]
```diff
@@ -239,8 +239,8 @@
  * non-system fonts are only applied once loaded in order to
  * prevent FOIT.
  */
-$font-default-sans-fallback: 'Helvetica Neue', Helvetica, sans-serif;
-$font-default-serif-fallback: Georgia, serif;
+ $font-default-sans-fallback: -apple-system, 'Helvetica Neue', Helvetica, 'Hiragino Sans', sans-serif;
```
Comment (hallee) [inline scss/fonts.scss:242]:
```suggestion
$font-default-sans-fallback: -apple-system, 'Helvetica Neue', Helvetica, 'Hiragino Sans', sans-serif;
```

**Review by hallee — DISMISSED** [review]

**Review by hallee — APPROVED** [review]

**Pull request merged**